CHANGELOG.md

```diff
@@ -2,12 +2,16 @@
 
 [Full changelog](https://github.com/mozilla/glean/compare/v53.1.0...main)
 
+* General
+* Rust
+  * The Ping Rate Limit type is now accessible in the Rust Language Binding ([#2528](https://github.com/mozilla/glean/pull/2528))
+  
 # v53.1.0 (2023-06-28)
 
 [Full changelog](https://github.com/mozilla/glean/compare/v53.0.0...v53.1.0)
 
 * General
-  * Gracefully handle the waiting thread going away during shutdown ([#2503](https://github.com/mozilla/glean/pull/2503))
+  * Gracefully handle the waiting thread going away during shutdown ([#2503](/2503))
```

Comment:
Mistakenly modified this link?

Reply:
Hmm appears so good catch. Not sure why that happened =/